lib/views/index.erb

```diff
@@ -0,0 +1,20 @@
+<% if current_user %>
+	<div id='new-peep'>
+  		<form action='/peeps' method='post'>
+    		<input type='text' size="40" style="font-size: 16px" name='message'>
+   			<input type='submit' value='Peep'>
+  		</form>
+	</div>
+<% end %>
+<section class="signupbox">
+	<main>
+		<ul>
+			<% @peeps.each do |peep| %>
+				<div class="peep">
+    				<li class="name"><%= peep.name %></li> <li class="username">@<%= peep.username %></li> <li class="time"><%= peep.time.strftime("%I:%M%p") %></li>
+    				<article><%= peep.message %></article>
+    			</div>
+  			<% end %>
+		</ul>
+	</main>
+</section>
```

Comment:
Indentation on your erb files is weird - set it up to expand to spaces, either 2 or 4 (preferably 2)